src/datasources/transaction-api/transaction-api.service.ts

```diff
@@ -804,6 +804,8 @@ export class TransactionApi implements ITransactionApi {
         cacheDir,
         url,
         this.defaultNotFoundExpirationTimeSeconds,
+        undefined,
+        this.defaultExpirationTimeInSeconds,
```

Comment:
We need a feature flag for this one for one reason: if we deploy a production version of the CGW before the hooks are on production on the Transaction Service then the users will not see new messages until the cache entry expires.

Reply:
Good point, I was thinking of releasing this after the release in the Transaction Service, but maybe is more solid if we deliver this behind an FF, even if there's some work for cleaning after that, to cover the scenario where the Transaction Service needs to rollback for some reason after its deployment. 

I'll implement the FF for this 👍